src/cocotb/handle.py

```diff
@@ -1046,6 +1058,12 @@ def value(self) -> float:
     def value(self, value: float) -> None:
         self.set(value)
 
+    @deprecated(
+        "`float()` casts on handles have been deprecated. Use `str(handle.value)` instead."
```

Comment:
```suggestion
        "`float(handle)` casts have been deprecated. Use `float(handle.value)` instead."
```
This one had a real error: `s/str/float/`